staking/tests/max_pos.ts

```diff
@@ -81,6 +81,9 @@ describe("fills a stake account with positions", async () => {
       ) {
         await provider.sendAndConfirm(transaction, [], {});
         transaction = new Transaction();
+        transaction.instructions.push(
+          ComputeBudgetProgram.setComputeUnitLimit({ units: 1_400_000 })
```

Comment:
this PR made me find this bug, previously we were just requesting compute units in the first tx